stor/dx.py

```diff
@@ -967,13 +1006,30 @@ def listdir(self, only='all', canonicalize=False):
 
     def listdir_iter(self, canonicalize=False):
         """Iterate the path as a dir, returning top-level directories and files.
+        If listing an empty path, lists the projects a user has access to.
 
         Args:
             canonicalize (bool, default False): if True, return canonical paths
 
         Returns:
             Iterable[DXPath]: Iterates over listed files directly within the resource
         """
+        if not self.project:  # list projects for empty path
+            kwargs = {
+                'describe': {'fields': {'name': True}},
+                'level': 'VIEW',
+                'explicit_perms': True
+            }
+            projects_generator = dxpy.find_projects(**kwargs)
```

Comment:
will this find public projects too?

Reply:
This will only list those public projects where the user was specifically given permission. So, if Myriad_Demo was a public project, it would show that in the list (because we were specifically invited into it) and won't show all other public projects.